Funcky.Test/Extensions/EnumerableExtensions/TakeEveryTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Funcky.Extensions;
+using Xunit;
+
+namespace Funcky.Test.Extensions.EnumerableExtensions
+{
+    public class TakeEveryTest
+    {
+        [Fact]
+        public void TakeEveryOnAnEmptySequenceReturnsAnEmptySequence()
+        {
+            var emptySource = Enumerable.Empty<string>();
+
+            Assert.Empty(emptySource.TakeEvery(5));
+        }
+
+        [Theory]
+        [InlineData(0)]
+        [InlineData(-1)]
+        [InlineData(-42)]
+        public void TakeEveryOnlyAcceptsPositiveIntervals(int negativeIntervals)
+        {
+            var numbers = new List<int> { 1 };
+
+            Assert.Throws<ArgumentOutOfRangeException>(() => numbers.TakeEvery(negativeIntervals));
+        }
+
+        [Fact]
+        public void TakeEverySelectsEveryNthElement()
+        {
+            var numbers = Enumerable.Range(-60, 120).ToList();
+
+            Assert.Equal(numbers.Count() / 6, numbers.TakeEvery(6).Count());
+
+            numbers.TakeEvery(6).ForEach(n => Assert.True(n % 6 == 0));
+        }
+
+        [Fact]
+        public void TakeEveryWithLessThanIntervalElementsReturnsOnlyFirstElement()
+        {
+            var numbers = new List<int> { 1, 2, 3, 4 };
+
+            Assert.Single(numbers.TakeEvery(4));
+            Assert.Equal(1, numbers.TakeEvery(4).Single());
```

Comment:
There's a `Assert.Single` overload that takes an expected value.
```suggestion
            Assert.Single(numbers.TakeEvery(4), expected: 1);
```